src/components/pad-view-only/document-infobar.tsx

```diff
@@ -0,0 +1,62 @@
+import moment from 'moment'
+import React from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { InternalLink } from '../common/links/internal-link'
+import { ShowIf } from '../common/show-if/show-if'
+import './document-infobar.scss'
+import {
+  DocumentInfoLineWithTimeMode,
+  DocumentInfoTimeLine
+} from '../editor/document-bar/document-info/document-info-time-line'
+
+export interface DocumentInfobarProps {
+  changedAuthor: string
+  changedTime: Date
+  createdAuthor: string
+  createdTime: Date
+  editable: boolean
+  noteId: string
+  viewCount: number
+}
+
+export const DocumentInfobar: React.FC<DocumentInfobarProps> = ({
```

Comment:
tbh I don't like this info lines. They are so codimd 1.6..  that's why we replaced them in the editor view with the document info modal. I would use that modal here, too.  (my opinion)

Reply:
You mean including the document-info button into the app-bar? That's a no-go in my opinion as it violates the document/app diffentiation.

Reply:
Of course, I only can talk for myself. But when someone sends me a link to a read-only-version of a HedgeDoc/CodiMD document, I like to see directly how up-to-date it is and who created or edited it last. Clicking another button just makes it more inconvenient to get that information.

In the editor, I can fully live with the button-solution. It even looks better because of not cluttering the UI with information everywhere. But the read-only-view is another thing. I almost would go as far as comparing it with a blog post or medium article - in both of those two you get presented the author and creation-time without further action.

To conclude: I wouldn't change that.